### PR TITLE
Feature/cache deauth acctsessionid

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Use CHI to cache the latest accounting request so we don't have to record accounting in the SQL tables for the deauthentication
# Impacts

That big beast that we call accounting
Deauthentication
# Delete branch after merge

YES
# NEWS file entries
## Enhancements
- Added option to deactivate the accounting recording in the SQL tables
- Added caching of the latest accounting request for use in access reevaluation.

### CODE REVIEW

**Comment by julsemaan** [comment]
Ready for review

File: lib/pf/accounting.pm [hunk 1]
```diff
@@ -74,6 +75,8 @@ Readonly our $ACCOUNTING_TRIGGER_RE => qr/
 Readonly our $DIRECTION_IN => 'IN';
 Readonly our $DIRECTION_OUT => 'OUT';
 
+our $CACHE = pf::CHI->new(namespace => "accounting");
```
Comment (rouzier) [inline lib/pf/accounting.pm]: Wrap creating the cache in a function.
This will cause problems in a threaded environment

**Comment by julsemaan** [comment]
As the frenchs would say : fixaide.

Re-review

**Comment by lzammit** [comment]
Une baguette et un croissant au beurre s'il vous plait*

**Pull request merged**